ci/taos/plugins-base/pr-audit-build-yocto.sh

```diff
@@ -78,9 +78,10 @@ function pr-audit-build-yocto-run-queue(){
     locale
     echo "[DEBUG] locale information: end   ---------------------------------------------"
 
-    # import environment variables from eSDK to use devtool command
-    # YOCTO_ESDK_ROOT="/var/www/kairos_sdk"
-    YOCTO_ESDK_ROOT="/var/www/poky_sdk"
+    # Import environment variables from eSDK to use devtool command
+    # For example, YOCTO_ESDK_ROOT="/var/www/kairos_sdk"
+    # Note that administrator of a server has to specify the location of eSDK at first.
+    YOCTO_ESDK_ROOT=""
```

Comment:
Should user set 'YOCTO_ESDK_ROOT' variable?

Reply:
NO. It is a role of CI Server `administrator`. 
So. it will not be moved to the ./config/ folder.